### DISCUSSION TITLE
Enable --squash for docker images

### DISCUSSION THREAD

**Discussion by keceli** [community/community#38106] [Actions]
Is it possible to use `--squash` for `docker` builds on GitHub actions? I tried enabling experimental features with `export DOCKER_CLI_EXPERIMENTAL=enabled`, but I still get: 

```bash
"--squash" is only supported on a Docker daemon with experimental features enabled
```

So, I guess docker daemon needs to be restarted. I was wondering if this would be enabled in the future, or if there is any workaround. Squashing could help reducing the size of the image and preventing leaks of secret keys.

**Comment by ossanna16** [comment]
Hi there @keceli and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
